lib/uffizzi/services/dev_service.rb

```diff
@@ -3,21 +3,50 @@
 require 'uffizzi/clients/api/api_client'
 
 class DevService
+  DEFAULT_REGISTRY_REPO = 'registry.uffizzi.com'
+  STARTUP_STATE = 'startup'
+  CLUSTER_DEPLOYED_STATE = 'cluster_deployed'
+
   class << self
     include ApiClient
 
-    DEFAULT_REGISTRY_REPO = 'registry.uffizzi.com'
+    def check_no_running_process!
+      if process_running?
+        Uffizzi.ui.say_error_and_exit("You have already started uffizzi dev. To stop the process do 'uffizzi dev stop'")
+      end
+    end
 
-    def check_running_daemon
-      return unless File.exist?(pid_path)
+    def check_running_process!
```

Comment:
This can be part of `stop_process` I think. Two methods with names`check_no_running_process!` and  `check_running_process!` is a little confusing.